examples/parallelCoordinates.html

```diff
@@ -12,6 +12,9 @@
         }
         svg {
             display: block;
+            float: left;
+            height: 450px !important;
+            width: 850px !important;
```

Comment:
I'd like to keep the example charts so that they expand with the window when you view just the example so we can test the responsiveness of the charts that way.  Is there a specific reason it has to be this size?